src/components/VideoList/VideoList.component.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import VideoItem from '../VideoItem';
+import { Wrapper } from './VideoList.styled';
+
+function VideoList({ items }) {
+  const hasItems = (input) => {
+    if (typeof input !== 'undefined' && input !== null && input !== 'undefined') {
+      console.log(items);
+      return true;
+    }
+    return false;
+  };
+  return (
+    <>
+      <Wrapper>
+        {!hasItems(items) && <p>No items found</p>}
+        {hasItems(items) && (
+          <>
+            {items.map((v) => (
+              <VideoItem item={v} key={v.etag} />
+            ))}
+          </>
+        )}
+      </Wrapper>
+    </>
```

Comment:
This line can be something like:

```jsx
<Wrapper>
  {!hasItems(items) && <p>No items found</p>}
  {hasItems(items) && items.map((v) => (
    <VideoItem item={v} key={v.etag} />
  ))}
</Wrapper>
```

the empty fragments are not necessary `<></>`